draft-ietf-quic-recovery.md

```diff
@@ -517,7 +517,7 @@ client, it is the client's responsibility to send packets to unblock the server
 until it is certain that the server has finished its address validation
 (see Section 8 of {{QUIC-TRANSPORT}}).  That is, the client MUST set the
 probe timer if the client has not received an acknowledgement for one of its
-Handshake or 1-RTT packets.
+Handshake or 1-RTT packets and the handshake has not been confirmed.
```

Comment:
As history, this text was written without HANDSHAKE_DONE in mind(because it hadn't been added) and I thought receiving a 1-RTT ACK was going to be how the handshake was confirmed.

Technically, receiving an ACK of 1-RTT confirms the handshake, so I prefer "or a HANDSHAKE_DONE frame." instead of "and the handshake has not been confirmed."  It's a bit more explicit, now that our mechanisms have stabilized.